gui/projectfile.h

```diff
@@ -557,6 +572,11 @@ class ProjectFile : public QObject {
     /** Check code in unused templates */
     bool mCheckUnusedTemplates;
 
+    /**
+     * @brief Usage of inline suppression.
```

Comment:
I suggest that "Usage" not used in this comment.. one of these alternatives would be better imho;

    Activate inline suppressions
    Enable inline suppressions